changelogs/fragments/2095_bugfix_vmware_guest_disk_round_size_to_int.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - vmware_guest_disk - round size to int, supporting float values properly
```

```suggestion
  - vmware_guest_disk - round size to int, supporting float values properly
    (https://github.com/ansible-collections/community.vmware/issues/123).
```

We should add an issue and / or PR to the changelog. This helps people to understand what has been changed an why.